CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Usage:
 * Fix #246: Dockerfile custom interpolation is broken
 * Fix #259: Cleanup unused properties inside Mojos
 * Fix #94: Properly define + document JKubeProjectAssembly behavior
+* Bump Fabric8 Kubernetes Client to v4.10.3
```

Comment:
Given you have to rebase to fix conflicts, please add the PR number here for future reference
```suggestion
* Fix #290: Bump Fabric8 Kubernetes Client to v4.10.3
```